jwt.js

```diff
@@ -1,23 +1,36 @@
 'use strict'
 
-const fp = require('fastify-plugin')
-const JWT = require('jsonwebtoken')
-const assert = require('assert')
+var fp = require('fastify-plugin')
+var JWT = require('jsonwebtoken')
+var assert = require('assert')
+var steed = require('steed')
 
-function fastifyJWT (fastify, opts, next) {
-  if (!opts.secret) {
+function wrapStaticSecretInCallback (secret) {
+  return function (_, __, cb) {
```

Comment:
Can you call these variable `request` and `reply`?

Reply:
Renaming to request and payload in next commit